source/_posts/potential-patent-infringement.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Link to   previous content is pending

Reply:
I am creating this blog for Abhinav and this is his first blog on our website so currently we dont have any links. I have added couple of links of his previous post in between content as per given in word doc by him.